cli/tests/it/cmd.rs

```diff
@@ -662,3 +666,97 @@ contract CTest is DSTest {
     let cache_after = fs::read_to_string(prj.cache_path()).unwrap();
     assert_eq!(cache, cache_after);
 });
+
+forgetest!(can_deploy_script_without_lib, |prj: TestProject, cmd: TestCommand| {
+    RuntimeOrHandle::new().block_on(async {
+        let port = next_port();
+        spawn(NodeConfig::test().with_port(port)).await;
+        let mut tester = ScriptTester::new(cmd, port, prj.root());
+
+        tester
+            .load_private_keys(vec![0, 1])
+            .await
+            .add_sig("BroadcastTestNoLinking", "deployDoesntPanic()")
+            .sim("SIMULATION COMPLETE. To broadcast these")
+            .execute("ONCHAIN EXECUTION COMPLETE & SUCCESSFUL")
+            .assert_nonce_increment(vec![(0, 1), (1, 2)])
+            .await;
+    })
+});
+
+forgetest!(can_deploy_script_with_lib, |prj: TestProject, cmd: TestCommand| {
+    RuntimeOrHandle::new().block_on(async {
+        let port = next_port();
+        spawn(NodeConfig::test().with_port(port)).await;
+        let mut tester = ScriptTester::new(cmd, port, prj.root());
+
+        tester
+            .load_private_keys(vec![0, 1])
+            .await
+            .add_sig("BroadcastTest", "deploy()")
+            .sim("SIMULATION COMPLETE. To broadcast these")
+            .execute("ONCHAIN EXECUTION COMPLETE & SUCCESSFUL")
+            .assert_nonce_increment(vec![(0, 2), (1, 1)])
+            .await;
+    });
+});
+
+forgetest!(can_resume_script, |prj: TestProject, cmd: TestCommand| {
+    RuntimeOrHandle::new().block_on(async {
```

Comment:
similar thing came up recently so, perhaps, useful to create a new `forgetest!` variant that expands to async test?

either new grammar `forgetest!(async <test>` or `forgetest_async!`